src/data.js

```diff
@@ -1117,6 +1117,27 @@ module.exports = [
       'Golang',
     ],
   },
+  {
+    name: 'Rikin Patel',
+    description: 'Experienced Front-end developer, Passionate about Javascript',
+    url: 'https://patelrikin.com/uses',
```

Comment:
This just redirects to the root of your site.

Feel free to reopen a PR when you've got a uses page

Reply:
Added uses section on my website. I dont have dedicated page as its a single page layout but you can see in the footer section stuff that I use including website techstack that it was made of.

Reply:
created new PR: https://github.com/wesbos/awesome-uses/pull/546 since I cant reopen this one.